Audacia.Log.AspNetCore/README.md

```diff
@@ -114,9 +145,9 @@ To configure the overrides for "sub" and "role" add the `LogActionFilter` sectio
 }
 ```
 
-##### Configure global request filtering
+##### Configure global request and response filtering

```

Comment:
This section should mention it also configures the dependency tracking logs, unless you add a new section/type in which case that should be documented too

Reply:
Have split it out the config, and added a new section for configuring the dependencies. Can this stay the same?